src/main/java/com/google/api/codegen/viewmodel/DescriptorConfigView.java

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import javax.annotation.Nullable;
+
+@AutoValue
+public abstract class DescriptorConfigView implements ViewModel {
+  public abstract String templateFileName();
+
+  public abstract List<PageStreamingDescriptorView> pageStreamingDescriptors();
+
+  @Nullable
```

Comment:
Why is this field nullable, but all others are not and use an empty list?

Reply:
I followed the standard set [here]( 
https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/viewmodel/DynamicLangXApiView.java#L56-L57). I agree it does not seem there is a good reason. Should we update this and the DynamicLangXApiView?

Reply:
Yes, lets do that. If you find that removing the nullability from the DynamicLangXApiView class, then leave as-is